app/src/main/res/values-pl-rPL/strings.xml

```diff
@@ -0,0 +1,142 @@
+    <string name="resume_scroll">Wznów przewijanie</string>
+    <string name="uploading_image">Prześlij zdjęcie</string>
+    <string name="settings">Ustawienia</string>
+    <string name="nuuls_upload_disclaimer">Zewnętrzny hosting dostarczony przez https://nuuls.com/i, używaj na własną odpowiedzialność.</string>
+    <string name="nuuls_upload_title">Niestandardowe przesyłanie plików</string>
+    <string name="snackbar_message_copied">Skopiowano wiadomość</string>
+    <string name="snackbar_error_copied">Skopiowano wiadomość błędu</string>
+    <string name="notification_stop">Stop</string>
+    <string name="notification_title">FeelsDankMan</string>
+    <string name="notification_message">DankChat działa w tle</string>
+    <string name="emote_menu_hint">Otwórz menu emotek</string>
+    <string name="login_title">Zaloguj się do Twitch.tv</string>
+    <string name="hint_connected">Napisz wiadomość</string>
+    <string name="hint_disconnected">Rozłączono</string>
+    <string name="hint_not_logged_int">Nie jesteś zalogowany</string>
+    <string name="notification_new_mentions">Masz nowe wzmianki</string>
+    <string name="notification_mention">%1$s wspomniał o Tobie w #%2$s</string>
+    <string name="notification_notify_mention">Zostałeś wspomniany w #%1$s</string>
+    <string name="snackbar_login">Logowanie jako %1$s</string>
+    <string name="snackbar_login_failed">Nie udało się zalogować</string>
+    <string name="snackbar_image_uploaded">Skopiowano: %1$s</string>
+    <string name="snackbar_upload_failed">Błąd podczas przesyłania</string>
+    <string name="snackbar_upload_failed_cause">Błąd podczas przesyłania: %1$s</string>
+    <string name="snackbar_retry">Ponów</string>
+    <string name="snackbar_data_reloaded">Przeładowano emotki</string>
+    <string name="snackbar_data_load_failed_cause">Błąd podczas ładowania danych: %1$s</string>
+    <string name="add_channel_hint">Nazwa kanału</string>
+    <string name="emote_menu_tab_subs">Subskrybcje</string>
+    <string name="emote_menu_tab_channel">Kanał</string>
+    <string name="emote_menu_tab_global">Globalne</string>
+    <string name="system_message_connected">Połączono</string>
+    <string name="system_message_disconnected">Rozłączono</string>
+    <string name="system_message_no_history">Nie widzisz historii wiadomości ponieważ zrezygnowałeś z tej usługi.</string>
+    <string name="timed_out_message">&lt; Wiadomość usunięta ></string>
+    <string name="multi_entry_header_regex">Regex</string>
+    <string name="multi_entry_header_match_user">Sprawdzaj nazwy użytkowników</string>
+    <string name="multi_entry_add_entry">Dodaj wpis</string>
+    <string name="multi_entry_remove_content_description">Usuń wpis</string>
+    <string name="multi_entry_hint">Wpis</string>
+    <string name="login_expired">Logowanie wygasło! Spróbuj zalogować się ponownie.</string>
+    <string name="message_history_disclaimer_title">Message history disclaimer</string>
```

Comment:
Not translated to polish
Suggested translation: "Informacja o historii wiadomości"

Reply:
Fixed in `eb7f80b`